subprojects/core-api/src/main/java/org/gradle/api/file/CopySpec.java

```diff
@@ -379,6 +379,13 @@
     @Override
     CopySpec expand(Map<String, ?> properties);
 
+    /**
+     * {@inheritDoc}
+     * @since 7.2
```

Comment:
I'm not sure, is the `@since` required here?

Reply:
Maybe I just overlooked it while moving code around or `sanityCheck` was complaining. It is unnecessary, so I removed it.